src/main/java/uk/gov/ida/verifyserviceprovider/factories/saml/AuthnRequestFactory.java

```diff
@@ -61,12 +70,22 @@ public AuthnRequestFactory(
     }
 
     public AuthnRequest build(LevelOfAssurance levelOfAssurance, String serviceEntityId) {
+        List<String> authnContexts = new ArrayList<>();
+        switch (levelOfAssurance) {
+            case LEVEL_1:
+                authnContexts.add(IdaAuthnContext.LEVEL_1_AUTHN_CTX);
```

Comment:
If we're using 'exact' then 'technically' I think we need to include LEVEL_2 aswell.

Reply:
The `switch` fall through should take care of this. See the two new tests in `AuthnRequestFactoryTest`.